src/main/java/me/tongfei/progressbar/ProgressState.java

```diff
@@ -67,6 +74,8 @@ synchronized long getMax() {
     // The progress, normalized to range [0, 1].
     synchronized double getNormalizedProgress() {
         if (max <= 0) return 0.0;
+        else if (max - start < current)
```

Comment:
I don't follow the logic here -- what condition does `max - start < current` check?